cpp/src/io/parquet/predicate_pushdown.cpp

```diff
@@ -388,40 +388,17 @@ class stats_expression_converter : public ast::detail::expression_transformer {
 };
 }  // namespace
 
-std::optional<std::vector<std::vector<size_type>>> aggregate_reader_metadata::filter_row_groups(
+std::tuple<std::optional<std::vector<std::vector<size_type>>>, size_type, size_type>
```

Comment:
would you mind turning this into a struct? It seems a bit too much to keep in mind what these represent